packages/cli/package.json

```diff
@@ -36,6 +36,8 @@
   "homepage": "https://github.com/bhovhannes/bundle-validator#readme",
   "dependencies": {
     "commander": "8.3.0",
+    "cosmiconfig": "^7.0.1",
+    "fast-glob": "^3.2.7",
```

Comment:
I think you need to run `yarn` in the root directory to update `yarn.lock`. I think that's why tests are failing on CI.

Also, please remove `^`. Having exact versions is more predictable and secure.